qa/workunits/rbd/cli_generic.sh

```diff
@@ -743,6 +743,14 @@ test_migration() {
     rbd migration execute test1
     rbd migration commit test1
 
+    # testing trash
+    rbd migration prepare test1
+    rbd trash mv test1 2>&1 | grep 'cannot move migrating image to trash'
```

Comment:
I am confused by this: it throws an error saying you cannot move a migrating image to the trash but it does move it to the trash?

Reply:
I don't think "it does move it to the trash", image of 'test1' in trash is the source image (trash moved at migration prepare proceed) and this 'test1' is the destination image (could not move to trash).